{"package_name": "java.nio.file", "subclass_list": [], "class_description": "Runtime exception thrown if an I/O error is encountered when iterating over\n the entries in a directory. The I/O error is retrieved as an IOException using the getCause() method. ", "class_type": "class", "typealias": [], "interface_list": [""], "class_inherit_list": ["ConcurrentModificationException"], "Methods": [{"params": [{"param_name": ["cause"], "param_type": "IOException", "param_description": " the IOException that caused the directory iteration\n          to fail"}], "class_name": "DirectoryIteratorException", "is_static": false, "return_value": {"return_name": "", "return_description": [], "return_type": []}, "method_name": "init", "method_description": "Constructs an instance of this class.\n", "method_declar": ""}, {"params": [], "class_name": "DirectoryIteratorException", "is_static": false, "return_value": {"return_name": [], "return_description": ["the cause"], "return_type": ["IOException"]}, "method_name": "getCause", "method_description": "Returns the cause of this exception.\n", "method_declar": ""}], "class_name": "DirectoryIteratorException", "Vars": null}